### ISSUE TITLE
Issue with Upgrading pip to the latest version

### ISSUE THREAD

**Issue by neranga6** [boltgolt/howdy#960]
I'm having following Python pip issue when tried to install howdy..  Please help

<pre>Unpacking howdy (2.6.1) ...
Setting up howdy (2.6.1) ...
/var/lib/dpkg/info/howdy.postinst:145: SyntaxWarning: invalid escape sequence &apos;\
w&apos;
  &quot;davisking-dlib-\w+/(dlib/(http_client|java|matlab|test/)|&quot;

&gt;&gt;&gt; <font color="#26A269">Upgrading pip to the latest version</font>

<font color="#C01C28"><b>error</b></font>: <b>externally-managed-environment</b>

<font color="#C01C28">×</font> This environment is externally managed
<font color="#C01C28">╰─&gt;</font> To install Python packages system-wide, try apt install
<font color="#C01C28">   </font> python3-xyz, where xyz is the package you are trying to
<font color="#C01C28">   </font> install.
<font color="#C01C28">   </font> 
<font color="#C01C28">   </font> If you wish to install a non-Debian-packaged Python package,
<font color="#C01C28">   </font> create a virtual environment using python3 -m venv path/to/venv.
<font color="#C01C28">   </font> Then use path/to/venv/bin/python and path/to/venv/bin/pip. Make
<font color="#C01C28">   </font> sure you have python3-full installed.
<font color="#C01C28">   </font> 
<font color="#C01C28">   </font> If you wish to install a non-Debian packaged Python application,
<font color="#C01C28">   </font> it may be easiest to use pipx install xyz, which will manage a
<font color="#C01C28">   </font> virtual environment for you. Make sure you have pipx installed.
<font color="#C01C28">   </font> 
<font color="#C01C28">   </font> See /usr/share/doc/python3.12/README.venv for more information.

<font color="#A347BA"><b>note</b></font>: If you believe this is a mistake, please contact your Python installation 
or OS distribution provider. You can override this, at the risk of breaking your
 Python installation or OS, by passing --break-system-packages.
<font color="#2AA1B3"><b>hint</b></font>: See PEP 668 for the detailed specification.
<font color="#C01C28">Error while running last command</font>
<b>dpkg:</b> error processing package howdy (--configure):
 installed howdy package post-installation script subprocess returned error exit
 status 1
Processing triggers for man-db (2.12.0-4build2) ...
Errors were encountered while processing:
 howdy
<font color="#C01C28"><b>E: </b></font>Sub-process /usr/bin/dpkg returned an error code (1)</pre>

**Comment by SvetoslavTr** [comment]
Same here.

**Comment by almaron** [comment]
Yup, still same. System-wide package installation has been disallowed on latest python versions without that flag

**Comment by Jawkneebee** [comment]
Same here just tried installing this on Linuxmint 22

**Comment by TaliBytes** [comment]
As of 12/11/24 this is still an issue. Linux Mint.
![howdy-failed-install-img](https://github.com/user-attachments/assets/0a0e3d3e-42e2-4d02-a699-94eb41f11cd4)

**Comment by krickert** [comment]
Same issue here.  There might be a workaround in the forums

**Comment by anonymouspenguin1024** [comment]
Same issue. Apparently howdy' installation script needs to be updated to reflect this change.

**Comment by carloshuete** [comment]
Same issue with Linux Mint

**Comment by MountainMaster** [comment]
Same issue with TUXEDO OS based on Ubuntu 24.04

![image](https://github.com/user-attachments/assets/58f57c41-5007-4b63-976d-9559b3a7ce7c)

**Comment by Hamster1962** [comment]
I have an issue too. Not exactly the same but close, a Python problem.
Unfortunately it would seem the developer has disappeared.

**Comment by boltgolt** [comment]
Maybe not disappeared but I definitely have way way less time to spend on Howdy now and large changes are needed. I'll look at PRs but I'm not sure I can contribute much myself anymore

**Comment by MountainMaster** [comment]
This looks like a workaround with a unofficial PPA. Did not test it yet. Maybe the changes done there can be used to fix the main repo.

https://ubuntuhandbook.org/index.php/2024/10/howdy-ubuntu-2404/amp/

**Comment by rong-xiaoli** [comment]
> This looks like a workaround with a unofficial PPA. Did not test it yet. Maybe the changes done there can be used to fix the main repo.
> 
> https://ubuntuhandbook.org/index.php/2024/10/howdy-ubuntu-2404/

May I ask is Ubuntu deb package also available for Debian? Or are there any restriction? Thanks.

**Comment by ankitsm08** [comment]
Same here

**Comment by skjerns** [comment]
howdy remains more or less broken because of this. too bad :( such a useful program
